scripts/build-windows.ps1

```diff
@@ -0,0 +1,101 @@
+# INTRO:
+# This file is used to build ferdium on both x64 and arm-based macos. It also handles any corrupted node modules with the 'CLEAN' env var (set it to 'true' for cleaning)
```

Comment:
This should be changed to refer to windows with powershell (so some minimum win version?) Do we also need to specify the version of PS?